### PR TITLE
fixed zero division error

### PR DESCRIPTION
**Pull request by serkakres**

### CODE REVIEW

File: commands.sh [hunk 1]
```diff
@@ -28,10 +29,16 @@ function getRepoAge
 commitCount=$(git rev-list HEAD --count)
 fileCount=$(git ls-files | wc -l)
 
+if [ "$totalDays" -eq "0" ]; then
```
Comment (d4rkd3v1l) [inline commands.sh:31]: good catch

File: commands.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
-source "git-stats.widget/config.sh"
+
+source "GitStats/config.sh"
```
Comment (d4rkd3v1l) [inline commands.sh]: Why did you change the name/path?

File: index.coffee [hunk 3]
```diff
@@ -1,40 +1,41 @@
...
-  bottom 0
+  top 60px
   padding 15px
   margin 0
   background rgba(#000, 0.5)
   -webkit-backdrop-filter: blur(30px)
 
   font-family AdelleLight
   font-size 1em
-  color #52d089
```
Comment (d4rkd3v1l) [inline index.coffee]: Why did you "remove" the colors?

**Comment by serkakres** [comment]
oh sorry, these changes were only for me, but i have forgotten to check out new branch :( corrected now.
